components/retro-go/targets/t-deck-plus/env.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+os.environ["IDF_TARGET"] = "esp32s3"
+os.environ["FW_FORMAT"] = "t-deck-plus"
```

Comment:
This should be "none" if no .fw is supported.

Reply:
I didn't understand how .fw works, maybe that can be added?

Reply:
Yeah esplay-based and odroid-based have a factory partition software to flash the main firmware from the sd card. 

You could port https://github.com/ducalex/odroid-go-multi-firmware to the t-desk if you're interested in such feature (but maybe there's a better option than my version, it's been years since I've looked into it).